### DISCUSSION TITLE
Tests with docker image on local repository first

### DISCUSSION THREAD

**Discussion by bilak** [renovatebot/renovate#17338] [Request Help] [answered]
Hello, 
I'd like to test some configurations without pushing them to the gitlab repository. Is that somehow possible with [docker images](https://github.com/renovatebot/docker-renovate-full)? I've seen [this document](https://github.com/renovatebot/renovate/blob/main/docs/development/local-development.md) but it's not clear to me if it's/isn't possible. 

Why I'd like to test it like this? because I don't have full access rights to the gitlab config and I'm not allowed to change some parts.

Thanks

**Comment by viceice** [comment, marked as answer]
not possible

**Reply by bilak** [reply]
thanks for quick response. any plan to support this in future?

**Reply by viceice** [reply]
not yet
